evm_arithmetization/tests/add11_yml.rs

```diff
@@ -166,12 +169,291 @@ fn add11_yml() -> anyhow::Result<()> {
     };
 
     let mut timing = TimingTree::new("prove", log::Level::Debug);
-    let proof = prove::<F, C, D>(&all_stark, &config, inputs, &mut timing, None)?;
+    let max_cpu_len = 1 << 20;
+    let proof = prove::<F, C, D>(
+        &all_stark,
+        &config,
+        inputs,
+        max_cpu_len,
+        0,
+        &mut timing,
+        None,
+    )?;
     timing.filter(Duration::from_millis(100)).print();
 
     verify_proof(&all_stark, proof, &config)
 }
 
+#[test]
+#[ignore] // Too slow to run on CI.
+fn add11_segments_aggreg() -> anyhow::Result<()> {
+    init_logger();
```

Comment:
Nit: this is just a recursive copy of the above, maybe most of the code can be refactored and reused to reduce code duplication?